flink-end-to-end-tests/test-scripts/test_streaming_sql.sh

```diff
@@ -21,15 +21,14 @@ source "$(dirname "$0")"/common.sh
 
 TEST_PROGRAM_JAR=${END_TO_END_DIR}/flink-stream-sql-test/target/StreamSQLTestProgram.jar
 
-# copy flink-table jar into lib folder
-add_optional_lib "table"
-
 start_cluster
 $FLINK_DIR/bin/taskmanager.sh start
 $FLINK_DIR/bin/taskmanager.sh start
 $FLINK_DIR/bin/taskmanager.sh start
 
-$FLINK_DIR/bin/flink run -p 4 $TEST_PROGRAM_JAR -outputPath file://${TEST_DATA_DIR}/out/result
+PLANNER="${1:-old}"
```

Comment:
Move the argument definition to the beginning of the file, just as in other tests.

Reply:
Agree